src/ui/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine:3.9
+
+
+ENV APP_HOME /app
+RUN mkdir $APP_HOME
+
+WORKDIR $APP_HOME
+COPY Gemfile* $APP_HOME/
+COPY . $APP_HOME
```

Comment:
Воу-воу, крайний alpine, да еще и конкретные версии софта указаны =)

Reply:
Hadolint утверждает, что классные парни ставят версии:

[hadolint] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>` (DL3018)

Не стал с ним спорить :)

Reply:
Понял-понял =)